### PR TITLE
Add Belgium shields

### PR DESCRIPTION
**Pull request by Blijbol**
Probably the most boring country? Only the N-road shields actually differ from the default shields of this project.

I skipped the Bretelles (B-roads) because none have been mapped yet. Of course these too look the same as default shields anyway.

Photos of road signs: https://www.wegenwiki.nl/Bewegwijzering_in_Belgi%C3%AB

Schermafdruk van 2022-06-12 17-02-58

### CODE REVIEW

File: style/js/shield_defs.js [hunk 1]
```diff
@@ -2529,6 +2529,20 @@ export function loadShields(shieldImages) {
...
+  );
+  shields["BE:N-road"] = roundedRectShield(
+    Color.shields.blue,
+    Color.shields.white
+  );
+  shields["BE:R-road"] = roundedRectShield(
+    Color.shields.white,
+    Color.shields.black
+  );
+
```
Comment (claysmalley) [inline style/js/shield_defs.js:2545]: Let's keep this section sorted by country code. So `be` would go between `ba` and `bg`.

Comment (Blijbol) [reply style/js/shield_defs.js]: My bad, fixed in force-push.

**Review by claysmalley — APPROVED** [review]
Dank u (et merci)!

**Comment by ZeLonewolf** [comment]
Shields that are intentionally black on white should still be explicitly declared, especially now that we are considering #402 which proposes to change the default fallback shield. This can happen in this PR or in a separate one.

**Comment by Blijbol** [comment]
@ZeLonewolf This PR does indeed declare them. (except for the not-yet-mapped B-roads)

**Pull request merged**